.github/workflows/node.js.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
```

Comment:
Hopefully addressing the comments in the other MR will allow this to work

Reply:
@glenn-hamilton I've tried rebasing but still not running ?

Reply:
Seems it has 3 errors https://github.com/omcmenamin/Belfast_Team2_Frontend/actions/runs/6247497427 around issues with packages and tests

Reply:
@glenn-hamilton sorted now